Makefile.in

```diff
@@ -14,26 +14,30 @@
 # distribution  cleans and builds a .tgz
 # tags          builds etags file from all .c and .h files
 
-USE_OPENSSL = @USE_OPENSSL@
+CRYPTO_LIBDIR = @CRYPTO_LIBDIR@
+USE_EXTERNAL_CRYPTO = @USE_EXTERNAL_CRYPTO@
 HAVE_PCAP = @HAVE_PCAP@
 
+FIND_LIBRARIES = DYLD_LIBRARY_PATH=$(CRYPTO_LIBDIR)
```

Comment:
This won't be sufficient for Linux, where this needs to be LD_LIBRARY_PATH.  Why not export this value?